CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ CHANGELOG
   - Critical Update for Intel oneAPI DPC++/C++ Compiler: 2023.2.1
   - Critical Update for Intel Fortran Compiler & Intel® Fortran Compiler Classic: 2023.2.1
   - Remove `CommunicationParameters` from the Custom Slurm Settings deny list.
+- Add `DisableSudoAccessForDefaultUser` parameter to Disable Sudo Access of Default users in supported OSes.
```

Comment:
Let's remove the uppercase letters (eg to "Disable Sudo Access" => "to disable sudo access").
Also, every cluster has one single default user, so "Default users" => "default user" (singular)